### ISSUE TITLE
Issues to pass "Done Configuring Web Client"

### ISSUE THREAD

**Issue by KristofKlein** [microsoft/nav-docker#522]
Hi,

I want to start a BC Container (21.2W1-OnPrem). using SSL with isolation process. This currently leads on my machine to a stopped/stalling script execution that just stands still after outputting "Done Configuring Web Client". 

Checking this repo I can see that `Restart-WebAppPool -Name $AppPoolName -ErrorAction Ignore` is the command that comes next.

As the container is "up" but not BC I can connect to the same and run the command there again. Result is the same. Like all IIS operations are not working correctly. (e.g. get-IISAppPool) is also not returning and just keeps spinning... while the same execution on an isolation hyperV returns "instant". 

Also commands like "Get-NavServerInstance" are incredible slow. others like Get-NavCompany instead are fast as usal.

I had several of those containers up with isolation. So it is the needle in the haystack again. As a last resort I zap'd the docker installation. But the error is persisting... 

no difference if host powershell runs as admin or user.

About the hosts: 
Version: 21.2.49946.49990
Platform: 21.0.49939.49984
Generic Tag: 1.0.2.14
Container OS Version: 10.0.19041.1415 (2004)
Host OS Version: 10.0.19045.2604 (22H2)

an iisrestart.exe gives me this:
iisreset.exe

Attempting stop...
Restart attempt failed.
The system cannot find the file specified. (2147942402, 80070002)

Compare HyperV vs. Process I can see in the event log that at the moment it stops for process the hyperV does some event log entries around IISASPNetCore Module V2 "Sent shutdown HTTP " and a bit later "Application '/LM/W3SVC/1331555702/ROOT/BC' started"

Any advice/idea on this behavior? or how to tackle it down?

Mvh
Kristof

**Comment by KristofKlein** [comment]
As you are away (I think DoKUK?), I went a bit further with this and tried the following: docker run -it mcr.microsoft.com/windows/servercore:10.0.19041.1415 powershell.exe
inside of the container (running in Process Isolation) I executed install-windowsfeature for IIS components. that works fine, and also the execution of e.g. get-iisapppool returns instant ... of course this is a lame compare...but something happens to the container that my system does not like.

also a docker run -it mcr.microsoft.com/businesscentral:10.0.19041.1415 powershell.exe works fine including the get-iisapppool.

Something goes on in the container spin up / applying artifacts that does not work for me...

**Comment by KristofKlein** [comment]
As you are away (I think DoKUK?), I went a bit further with this and tried the following: docker run -it mcr.microsoft.com/windows/servercore:10.0.19041.1415 powershell.exe
inside of the container (running in Process Isolation) I executed install-windowsfeature for IIS components. that works fine, and also the execution of e.g. get-iisapppool returns instant ... of course this is a lame compare...but something happens to the container that my system does not like.

also a docker run -it mcr.microsoft.com/businesscentral:10.0.19041.1415 powershell.exe works fine including the get-iisapppool.

Something goes on in the container spin up / applying artifacts that does not work for me...

**Comment by KristofKlein** [comment]
Back to the issue: So after I gave up and came in the morning again, the faulty container tried to start ( and failed with the BC part of course). Still I logged into the container and I tried to execute the get-issapppool command. and it worked. instanted. "Strange, strange". So I did another container with isolation and ssl... and... it worked...
So what is the catch?! A dangling windows update maybe? Beeing in a different network (work vs. HomeOffice)? Just because it is friday?

That is why docker once in a while makes me run away from computers...

So never mind! Hope you enjoyed your DoKUK :D
